garden/tutorials/actors/actor_demo.sdf

```diff
@@ -1,21 +1,21 @@
-<sdf version="1.8">
+<sdf version="1.9">
   <world name="actors">
     <plugin
-      filename="libignition-gazebo-physics-system.so"
-      name="ignition::gazebo::systems::Physics">
+      filename="libgz-sim-physics-system.so"
```

Comment:
You can remove `lib` and `.so`
```suggestion
      filename="gz-sim-physics-system"
```

Reply:
Thanks, yeah, not sure how I missed that.